boards/posix/native_posix/doc/board.rst

```diff
@@ -366,10 +384,66 @@ If the SW unmasks a pending interrupt while running, or triggers a SW
 interrupt, the interrupt controller may raise the interrupt immediately
 depending on interrupt priorities, masking, and locking state.
 
-Normally the resulting executable runs fully decoupled from the real host time.
-That is, simulated time will advance as fast as it can. This is desirable when
-running in a debugger or testing in batch, but not if one wants to interact
-with external interfaces which are based on the real host time.
+About time in native_posix
+==========================
+
+Normally simulated time advances runs fully decoupled from the real host time
+and as fast as the host compute power would allow.
+This is desirable when running in a debugger or testing in batch, but not if
+one wants to interact with external interfaces which are based on the real
```

Comment:
change to:

    but not if interacting with external interfaces based on the real host time.